test/index.spec.ts

```diff
@@ -144,6 +145,43 @@ describe("addLambdaFunctions", () => {
       ],
     });
   });
+
+  it("adds aws-lambda-nodejs NodejsFunctions along with aws-lambda Functions", () => {
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app, "stack", {
+      env: {
+        region: "sa-east-1",
+      },
+    });
+    const hello1 = new lambda.Function(stack, "HelloHandler1", {
+      runtime: lambda.Runtime.NODEJS_10_X,
+      code: lambda.Code.fromInline("test"),
+      handler: "hello.handler",
+    });
+    const hello2 = new lambdaNodeJS.NodejsFunction(stack, "HelloNodeJsHandler", {
+      runtime: lambda.Runtime.NODEJS_14_X,
+      entry: "./test/lambda/example-lambda.ts",
+      handler: "handler",
+    });
+
+    let threwError = false;
+    try {
+      const datadogCdk = new Datadog(stack, "Datadog", {
+        nodeLayerVersion: 20,
+        pythonLayerVersion: 28,
+        addLayers: true,
+        enableDatadogTracing: false,
+        flushMetricsToLogs: true,
+        site: "datadoghq.com",
+        forwarderArn: "forwarder-arn",
+        apiKey: "1234",
+      });
+      datadogCdk.addLambdaFunctions([hello1, hello2]);
+    } catch (e) {
+      threwError = true;
+    }
+    expect(threwError).toBe(false);
```

Comment:
I think this is a good test, but I think there is a slight gap between the title of the test versus what it is actually testing.

The title says "adds aws-lambda-nodejs NodejsFunctions along with aws-lambda Functions". In this case "add" means we are going to be applying the Datadog instrumentation to the function. However, the test actually seems to just check that calling this function with a `NodejsFunction` doesn't throw an error. That's good, but it doesn't actually prove that we added the Datadog instrumentation correctly. I would expand this test a bit to check that we actually applied the Datadog instrumentation. There are some other tests in this file that could give you some inspiration for this!

Reply:
Thanks for the tip! I updated the test to check that the stack with a NodejsFunction gets the updated layers.